Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV LANG=C.UTF-8 \
 
 RUN yum update -y \
   && yum install -y \
+    amazon-linux-extras install postgresql11 \
```

Comment:
This doesn't look right. Isn't this a separate command, not a list of things for `yum` to install?

Reply:
Indeed, from the build logs:

```
Package amazon-linux-extras-2.0.0-1.amzn2.noarch already installed and latest version
No package install available.
No package postgresql11 available.
No package netcat-openbsd available.
```

https://hub.docker.com/repository/registry-1.docker.io/itprotv/stack/builds/9f380509-bd19-44f5-a49e-3924aa6896e8

Reply:
> Indeed, from the build logs:
> 
> ```
> Package amazon-linux-extras-2.0.0-1.amzn2.noarch already installed and latest version
> No package install available.
> No package postgresql11 available.
> No package netcat-openbsd available.
> ```
> 
> https://hub.docker.com/repository/registry-1.docker.io/itprotv/stack/builds/9f380509-bd19-44f5-a49e-3924aa6896e8

Ah thanks, that on was easy to miss.

Reply:
I'm surprised that the install command succeeds when you try to install things that don't exist.

Reply:
same 🤔 I wonder if the issue is -y?

Reply:
Apparently this is just how `yum` works.

```
$ docker run --interactive --rm --tty amazonlinux:2
bash-4.2# yum install sudo does-not-exist
Loaded plugins: ovl, priorities
No package does-not-exist available.
...
bash-4.2# echo $?
0
```

😞

Reply:
https://serverfault.com/a/923357